sdk/documentintelligence/azure-ai-documentintelligence/README.md

```diff
@@ -896,6 +897,74 @@ The following add-on capabilities are available in this SDK:
 
 Note that some add-on capabilities will incur additional charges. See pricing: https://azure.microsoft.com/pricing/details/ai-document-intelligence/.
 
+### Get Raw JSON Result
+
+Can get the HTTP response by passing parameter `raw_response_hook`.
+<!-- SNIPPET:sample_get_raw_response.raw_response_hook -->
+
+```python
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.documentintelligence import DocumentIntelligenceAdministrationClient
+endpoint = os.environ["DOCUMENTINTELLIGENCE_ENDPOINT"]
+key = os.environ["DOCUMENTINTELLIGENCE_API_KEY"]
+
+client = DocumentIntelligenceAdministrationClient(
+    endpoint=endpoint, credential=AzureKeyCredential(key)
+)
+
+responses = []
+
+def callback(response):
+    response_status_code = response.http_response.status_code
+    response_body = response.http_response.json()
+    responses.append(response_status_code)
+    responses.append(response_body)
+
+with client:
```

Comment:
We didnt used to write sync samples that had `with client`, is this something we're doing across samples now?

Reply:
Checked other sync samples are not using `with` style, will make this one to be consistent.